met-web/src/components/engagement/view/EngagementWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { MetPaper } from 'components/common';
+import { Grid, Typography, Avatar, Link } from '@mui/material';
+
+const testData = {
+    image: 'https://cdn.pixabay.com/photo/2017/09/25/13/12/cocker-spaniel-2785074__340.jpg',
+    name: 'Remy Sharp',
+    email: 'TheJad@bc.gov.ca',
+    phone: '647-543-1332',
+};
+
+export default function EngagementWidget() {
+    return (
+        <Grid container alignItems="center" justifyContent="center" spacing={2} xs={12}>
+            <Grid item container xs={11}>
```

Comment:
What is the need for these 2 Grid at the beginning? Can't we remove them?

Reply:
I wanted to allocate 1/3 of the screen to the section and have the inner grid taking up 11/12 of that 1/3 so that the widget paper doesn't take up 100% of the space